frontend/src/components/messages/RequestConversations.vue

```diff
@@ -1,30 +1,29 @@
+<!-- eslint-disable vue/no-v-html -->
 <template>
   <v-container v-if="assistanceRequest">
     <v-row>
-      <v-col cols="9" class="d-flex align-center pl-0 pt-0 pb-4">
+      <v-col cols="7" lg="9" class="d-flex align-center pl-0 pt-0 pb-4">
         <span class="subject-header">Subject: {{ assistanceRequest.subject }}</span>
       </v-col>
-      <v-col cols="3" class="d-flex flex-column align-end pa-0">
-        <AppButton variant="text" v-if="assistanceRequest.isRead" @click="this.$emit('toggleMarkUnreadButtonInConversationThread')">
-          <v-icon class="icon" left>mdi-email-outline</v-icon>
+      <v-col cols="5" lg="3" class="d-flex flex-column align-end pa-0">
+        <AppButton v-if="assistanceRequest.isRead" size="small" :primary="false" class="conversations-button" @click="$emit('toggleMarkUnreadButtonInConversationThread')">
+          <v-icon left>mdi-email-outline</v-icon>
```

Comment:
it would be great if we can keep the distance between text and icon the same for all buttons.
![image](https://github.com/user-attachments/assets/383b7231-9afd-431e-b3ad-2ed2b109b14a)